ProjectStructureAndRoles.md

```diff
@@ -29,21 +31,25 @@ All Working Groups are documented in one GitHub repository “WorkingGroups” w
 
 As mentioned before the contributions to the Sub Projects were documented in the respective GitHub repositories of the Sub Projects, and the contributions to Working Groups were documented in the GitHub repository “WorkingGroups”. All contributions beyond that, esp. contributions to the Project itself or the the Steering Committee are documented in the GitHub repository “Governance”.
 
-**Use branches to reflect the artefact status**
+**Changes and contributions to CAMARA**
+
+In the Project the “Fork and pull model” is used. Changes and contributions to CAMARA shall follow this process:
+![PR_Approval_Process](/documentation/ressources/PR_Approval_Process.png)
+
+1 All changes / contributions shall start with an issue to which initial documents, API definitions or code can be attached. The issue should be discussed in the community before to work out the best form of action. By means of the issue the traceability of the contributions to CAMARA is also ensured. In addition as general rule there shall be no (later) pull request without an issue (except minor corrections, to be label as “minor”).
+
+2 Either the issue creator or one of the Sub Project Maintainers then shall assign the issue to one of the Contributors of the Sub Project.
 
-All Sub Projects and Working Groups have to work with 4 branches to reflect the status of the artefacts (API descriptions, API code, documentation, etc.):
-- *CON*	Branch for contributions
-- *WIP*	Branch for work-in-progress
-- *RTR*	Branch for artefacts which are ready-to-review
-- *main*	Branch for deliverables which can be shared externally
+3 This Contributor shall create a fork (if necessary), shall create a new branch, and shall copy the branch link into the issue. For the naming of the branch a clear and concrete description shall be used.
 
-All contributions (e.g. initial files, tools, templates) to CAMARA have to be made only to the CON branch. If Project participants intend to start working on an topic first they have to move the respective artefacts to the WIP branch. The original copy of the artefacts is deleted in the CON branch.
+Now he can work on the solution and add commits to the branch. If he’s done with the work the Contributor shall create a pull request, copy the pull request link into the issue and vice versa and add the commits to the pull request. For the naming of the pull request a clear and concrete description shall be used.
```

Comment:
It's enough to refer the addressed issue(s) in the pull request description. The link from issue to pull request is then done automatically by github:

```suggestion
Now they can work on the solution and add commits to the branch. If they are done with the work the Contributor shall create a pull request, refer the addressed issue(s) in the pull request description and add the commits to the pull request. For the naming of the pull request a clear and concrete description shall be used.
```